docs/sections/includeAndRender.md

```diff
@@ -0,0 +1,65 @@
+<link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# Command: include and render
+
+MarkBind provides `include` and `render` commands to allow you include the "included" fragments in a file and render the file into a HTML document.

```

Comment:
I wonder if include and render are really needed. As an author, I never use them. They are more useful for developers?